c-svc23Vectors/math.h

```diff
@@ -3,8 +3,9 @@
 
 #define EULER 2.71828
```

Comment:
irrelevant now?

Reply:
Tjaaaaa. The whole radial kernel part is irrelevant now I guess, so we could delete it. I don't see anything wrong with keeping it there though, as it is a nice argument for future works/experiments 😉